descriptors.tf

```diff
@@ -0,0 +1,23 @@
+# Unfortunately, cannot easily use plain format string
```

Comment:
Maybe add some pretext in one or a few sentences?
Because starting the file with "Unfortunately .." w/o providing context is confusing to read.